src/utilities/__tests__/printSchema-test.ts

```diff
@@ -779,7 +782,7 @@ describe('Type System Printer', () => {
         name: String!
         description: String
         args(includeDeprecated: Boolean = false): [__InputValue!]!
-        type: __Type!
+        type(nullability: __TypeNullability! = AUTO): __Type!
```

Comment:
The default mode should be Traditional since we don't want to expose unsuspecting clients to `SemanticallyNonNull` fields